tests/deposit_contract.t.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.6.0;
+pragma solidity ^0.6.2;
```

Comment:
Is there a warning associated with this?

Reply:
No, just the `{value: amount}` syntax was introduced with 0.6.2